src/app/Main.java

```diff
@@ -49,6 +52,8 @@ public static void main(String[] args) {
             throw new RuntimeException(e);
         }
 
+        SettingsButtonController settingsButtonController = SettingsPressedUseCaseFactory.createSettingsButtonController(viewManagerModel, settingsViewModel);
```

Comment:
This should not be created here, but rather in the LoggedInUseCaseFactory. Also, no need to add a separate use case factory specifically for settings pressed.